src/sentry/issues/streamline.py

```diff
@@ -0,0 +1,22 @@
+from sentry.models.organization import Organization
+from sentry.utils.options import sample_modulo
+
+
+def apply_streamline_rollout_group(organization: Organization):
+    # If unset, the organization's users can opt-in/out.
+    if organization.get_option("sentry:streamline_ui_only") is not None:
```

Comment:
do we need this check? wouldn't this option always be None at this point right after the org is created?